docs/model_config.md

```diff
@@ -73,20 +73,20 @@ description of the parameters below.
 
 | Name | Description |
 | :----------------------: | :-----------------------------: |
-| `triton_backend` | The backend to use for the model. Set to `tensorrtllm` to utilize the C++ TRT-LLM backend implementation. Set to `python` to utlize the TRT-LLM Python runtime. |
```

Comment:
Why would anyone ever use the python runtime today? Maybe this is needed for future runtime configuration, but at most should be an optional with default to `tensorrtllm`